platformio.ini

```diff
@@ -483,14 +483,15 @@ board = BigTree_SKR_Pro
 extra_scripts = pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
 build_flags = ${common.build_flags}
   -DUSBCON -DUSBD_USE_CDC -DUSBD_VID=0x0483 -DUSB_PRODUCT=\"STM32F407ZG\"
-  -DTARGET_STM32F4 -DSTM32F407_5ZX -DVECT_TAB_OFFSET=0x8000 -DHAVE_HWSERIAL6
+  -DTARGET_STM32F4 -DSTM32F407_5ZX -DVECT_TAB_OFFSET=0x8000 -DHAVE_HWSERIAL6 -DHAL_PCD_MODULE_ENABLED
 lib_deps =
   U8glib-HAL=https://github.com/MarlinFirmware/U8glib-HAL/archive/bugfix.zip
-  LiquidCrystal@1.3.4
-  TMCStepper@>=0.5.0,<1.0.0
+  LiquidCrystal
+  TMCStepper
```

Comment:
Don't remove the `<1.0.0` condition. It's there because of API compatibility and so that when I move on to 1.x.x releases, I don't have the burden of having to keep the interface backwards compatible.